lib/jinja.js

```diff
@@ -434,7 +434,7 @@ var jinja;
     var get = function() {
       var val, n = arguments[0], c = stack.length;
       while (c--) {
-        val = stack[c][n];
+        val = (n[0] == '^' ? window[n.slice(1)] : stack[c][n]);

```

Comment:
I don't think it's safe to assume that window exists. A strong use case is running in Node. I think we will need a better way to do `window[n.slice(1)]` part, perhaps a helper function `getGlobal(n.slice(1)`.